test/tojs.js

```diff
@@ -0,0 +1,20 @@
+var test = require('tape');
+var mobx = require('..');
+var observable = mobx.observable;
+var toJS = mobx.toJS;
+
+test('toJS should ignore HTMLElement', function(t) {
+	if (typeof document !== 'object') {
+		return t.end();
+	}
+
+	var image = observable(document.createElement('img'));
+
+	try {
+		toJS(image);
+	} catch (err) {
+		t.error(err);
```

Comment:
There is `t.doesNotThrow` for this https://github.com/substack/tape#tdoesnotthrowfn-expected-msg